### ISSUE TITLE
Question about plotting pareto front

### ISSUE THREAD

**Issue by YouHyin** [JonathanGuerne/BoTorchMutliObjective#1]
Hello, I am currently trying to implement the max-min multi-objective-bayesian optimization though botorch.
So your code is very helpful.
I have a question about plotting pareto front of max-min optimization problem.
![201479506-8e925678-8853-4ad5-a13b-31f93b61c72d](https://user-images.githubusercontent.com/113442395/218263334-a845498d-57fa-46a0-8217-5bf7988fbf44.png)
**I really want to plot pareto front plot like this figure!**

fig, axes = plt.subplots(1, 1, figsize=(7, 7))
algos = ["qNEHVI"]
cm = plt.cm.get_cmap('viridis')

batch_number = torch.cat(
    [torch.zeros(n_start), torch.arange(1, n_iter+1).repeat(n_samples, 1).t().reshape(-1)]
).numpy()
print(batch_number)
sc = axes.scatter(y[:, 0], y[:, 1], c=batch_number, alpha=0.8)

y_sorted = y[y[:, 0].sort()[1]]
print(y)
print(y_sorted)
axes.plot(
    **[_y[0] for non_dominated, _y in zip(is_non_dominated(-y_sorted), y_sorted) if non_dominated]**,
    [_y[1] for non_dominated, _y in zip(is_non_dominated(-y_sorted), y_sorted) if non_dominated],
    label="Pareto Front",
    c="r"
)

axes.set_xlabel("Objective 1")
axes.set_ylabel("Objective 2")
norm = plt.Normalize(batch_number.min(), batch_number.max())
sm =  ScalarMappable(norm=norm, cmap=cm)

sm.set_array([])
fig.subplots_adjust(right=0.9)
cbar_ax = fig.add_axes([0.93, 0.15, 0.01, 0.7])
cbar = fig.colorbar(sm, cax=cbar_ax)
cbar.ax.set_title("Iteration")

axes.legend()
plt.show()

**Question : I would really appreciate it if you could answer me with how to modify this code.**

Thank you !
